reliability/stub-core/src/main/kotlin/com/datadog/android/core/stub/StubSDKCore.kt

```diff
@@ -75,9 +75,9 @@ class StubSDKCore(
     /**
      * Returns the last metric is sent by [StubInternalLogger].
      */
-    fun lastMetric(): Map<String, Any>? {
+    fun lastMetric(): Map<String, Any> {
         return (internalLogger as StubInternalLogger)
-            .telemetryEventsWritten.lastOrNull { it["type"] == "mobile_metric" }
+            .telemetryEventsWritten.lastOrNull { it["type"] == "mobile_metric" } ?: emptyMap()
```

Comment:
very minor suggestion:

```suggestion
            .telemetryEventsWritten.lastOrNull { it["type"] == "mobile_metric" }.orEmpty()
```

Reply:
fixed